### ISSUE TITLE
Throw Timeout Error

### ISSUE THREAD

**Issue by codecomeon** [gronxb/webview-bridge#14]
If a webMethod throw a Error, and then you will get a Timeout Error.

```
/// WebMethod Define
async test() {
  throw new Error('test error');
}

/// App Invoke WebMethod
webMethod.current.test().catch(e => console.warn(e)); // then you will get a timeout error but not the "test error"
```

**Comment by gronxb** [comment]
Thank you very much for the report🥹. I will fix it and deploy it by today.

**Comment by gronxb** [comment]
Fixed in [v1.1.1](https://www.npmjs.com/package/@webview-bridge/web).

Thank you for your report. I always appreciate it 🥰
